clang/runtime/dpct-rt/include/util.hpp

```diff
@@ -830,6 +832,96 @@ class args_selector<n_nondefault_params, n_default_params, R(Ts...)> {
   }
 };
 
+enum class group_type {
+  work_group,
+  sub_group,
+  logical_group,
+  root_group
+};
+
+template<int dimensions = 3>
+class group_base
+{
```

Comment:
```suggestion
class group_base {
```

Reply:
Format style need to be unified.
Similar suggestions below.

Reply:
Updated